src/Microsoft.Framework.ConfigurationModel.Json/project.json

```diff
@@ -9,6 +9,10 @@
     "frameworks": {
         "net45": { },
         "dnx451": { },
-        "dnxcore50": { }
+        "dnxcore50": {
+            "dependencies": {
+                "System.Dynamic.Runtime": "4.0.10-*"

```

Comment:
Do we need to file a bug on this for Json.NET? And a bug for us to track removing this? This doesn't seem right.